ui/components/app/confirmation-warning-modal/confirmation-warning-modal.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
This component is currently not generic to any potential warning modal we may want to push for further friction on the confirmation page - it is specific to the confirmation of adding a custom network with chainId 1 via the API. I can make accept args/children and templatize the specific content if we decide this is the right approach, but because that's a rather annoying process I decided not to do it until I get some feedback confirming this approach in general.